src/storybook/pages/home/home-lean.ts

```diff
@@ -0,0 +1,143 @@
+import type { Args } from '@storybook/web-components';
+import type { TemplateResult } from 'lit';
+import { html } from 'lit';
+
+import '../../../elements/alert.js';
+import '../../../elements/container.js';
+import '../../../elements/action-group.js';
+import '../../../elements/checkbox.js';
+import '../../../elements/form-field.js';
+import '../../../elements/icon.js';
+import '../../../elements/paginator.js';
+import '../../../elements/radio-button.js';
+import '../../../elements/table.js';
+import { footer, navigation, skiplinkList } from './home.common.js';
+import './home.scss';
+
+export const homeLeanTemplate = (args: Args): TemplateResult => html`
+  <div>
+    ${skiplinkList()}
+
+    <sbb-header hide-on-scroll expanded>
+      <sbb-header-button id="hamburger-menu" icon-name="hamburger-menu-small" expand-from="small">
+        Menu
+      </sbb-header-button>
+      <div class="sbb-header-spacer"></div>
+      <sbb-header-link icon-name="magnifying-glass-small" href="/"> Search </sbb-header-link>
+      <sbb-header-button icon-name="user-small" class="sbb-header-shrinkable">
+        Sign in
+      </sbb-header-button>
+      <sbb-header-button icon-name="globe-small" id="language-menu-trigger" class="last-element">
+        English
+      </sbb-header-button>
+      <sbb-menu trigger="language-menu-trigger">
+        <sbb-menu-button aria-pressed="false">Deutsch</sbb-menu-button>
+        <sbb-menu-button aria-pressed="false">Français</sbb-menu-button>
+        <sbb-menu-button aria-pressed="false">Italiano</sbb-menu-button>
+        <sbb-menu-button icon-name="tick-small" aria-pressed="true"> English </sbb-menu-button>
+      </sbb-menu>
+      <a href="https://www.sbb.ch" slot="logo">
+        <sbb-logo protective-room="none"></sbb-logo>
+      </a>
+    </sbb-header>
+
+    ${navigation(true)}
+
+    <sbb-container expanded>
+      <section class="lean-section sbb-page-spacing-expanded">
```

Comment:
when using container, page spacing is controlled by container
```suggestion
      <section class="lean-section">
```